src/model_provider/model_provider.h

```diff
@@ -79,6 +80,73 @@ class FileModelProvider : public ModelProvider {
   std::string model_ = {};
 };
 
+
+namespace detail {
+
+inline std::string GenerateAutoserializeCmd(const std::string& module_path,
```

Comment:
I think we should have some validation wrt to the paths here before we use Python.

Reply:
What kind of path validation do you have in mind?

Reply:
If we can open the provided paths? Maybe it would be clearer to provide the error that way.

Reply:
It's already checked, here: https://github.com/triton-inference-server/dali_backend/blob/be81dbaf8801667d02a43d8d316bd13dc8898cf9/src/model_provider/model_provider.h#L67